drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -441,6 +450,23 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var forwardKeyboardHelpButtonTemplate = function(ctx) {
+        return h(
+            'button.unbutton.btn-default.btn-link.'+keyboardHelpButtonClass,
+            {attributes: {tabindex: 0}},
+            [
+                h(
+                    "span.btn-icon.fa.fa-keyboard-o",
+                    {attributes: {"aria-hidden": true}},
+                    []
+                ),
+                // appending space is the simplest way to avoid sticking text to the button, but also to have
+                // them underlined together on hover. When margin was used there was a gap in underlining
+                " " + keyboardHelpText
```

Comment:
Instead of concatenating the two strings, we can insert the space as a separate text node, which is a bit cleaner conceptually:

`h(..., [h('span' ...), " ", keyboardHelpText])`